tests/integration/targets/vmware_guest/tasks/check_mode.yml

```diff
@@ -49,11 +49,11 @@
   register: check_mode_state
```

Comment:
should we add more specific tests to verify check mode now that there is a more complete solution?

Reply:
While there are a lot of integration tests, they seem to test options mostly when creating a VM. It doesn't look like there are many that test _changing_ things. Well, I'll try to add at least a few additional tests for check mode.